engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -2473,22 +2484,13 @@ struct fiery_brand_t : public demon_hunter_spell_t
 
   fiery_brand_dot_t* dot_action;
 
-  fiery_brand_t( util::string_view name, demon_hunter_t* p, util::string_view options_str = {} )
-    : demon_hunter_spell_t( name, p, p->talent.vengeance.fiery_brand, options_str ),
-    dot_action( nullptr )
+  fiery_brand_t( demon_hunter_t* p, util::string_view options_str = {} )
+    : demon_hunter_spell_t( "fiery_brand", p, p->talent.vengeance.fiery_brand, options_str ),
+      dot_action( nullptr )
   {
     use_off_gcd = true;
 
-    /* NYI
-    if ( p->spec.fiery_brand_rank_2->ok() )
-    {
-      dot_action = p->get_background_action<fiery_brand_dot_t>( "fiery_brand_dot" );
-      if ( p->specialization() == DEMON_HUNTER_HAVOC || !from_demonic_oath )
-      {
-        dot_action->stats = stats;
-      }
-    }
-    */
+    dot_action = p->get_background_action<fiery_brand_dot_t>( "fiery_brand_dot" );
```

Comment:
Possible you may wanna merge these in the HTML report?
dot_action->stats = stats;